src/misc/pacer.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2020-2021  The DOSBox Staging Team
+ *  Copyright (C) 2020-2021  Kirk Klobe <kklobe@gmail.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+#include "pacer.h"
+
+#include <cmath>
+#include <string>
```

Comment:
I'm not a fan of implicit includes at all, is that our standard?

Reply:
There isn't a standard for this; and I agree depending on implicit includes from unrelated headers is a recipe to have things break.

My comment here is that pacer.h is Pacer's own header.. it's mandatory and not something that can go away on a whim..(which can happen for other headers). 

And because strings are part of pacer's API (by way of the constructor), the <string> include is upgraded out of pacer.cpp and into the front seat in pacer.h.  so when that happens in your class's own header, you can drop the duplicate entry on the CPP side. 
(So it's still explicit.. were not relying on random other headers to give us string. )

But if your prefer your classes to have it on both - I'm not going to sweat it 👍